cmd/system-probe/modules/network_tracer.go

```diff
@@ -80,11 +99,89 @@ type networkTracer struct {
 	restartTimer *time.Timer
 }
 
+func startRPCServer(tracer networkTracer) error {
+	socketFile := "/tmp/my_grpc.sock"
+
+	// Remove existing socket file if it exists
+	os.Remove(socketFile)
+
+	listener, err := net.Listen("unix", socketFile)
+	if err != nil {
+		return err
+	}
+
+	server := grpc.NewServer()
+	test2.RegisterSystemProbeServer(server, &tracer)
+
+	useGRPCServer := dd_config.SystemProbe.GetBool("service_monitoring_config.use_grpc")
+	if useGRPCServer {
+		go func() {
+			log.Info("gRPC server listening on Unix domain socket...")
+
+			err = server.Serve(listener)
+			if err != nil {
```

Comment:
```suggestion
			if err := server.Serve(listener); err != nil {
```

Reply:
if you're not logging the error, you should just ignore it
I do suggest to add visibility on errors